src/components/implementation/tests/unit_cxx/Makefile

```diff
@@ -0,0 +1,18 @@
+C_OBJS=cxx.o
+COMPONENT=cxx_test.o
+INTERFACES=
+DEPENDENCIES=cbuf_mgr printc sched valloc mem_mgr_large torrent evt timed_blk lock
+IF_LIB=./libstdc++.a ./libsupc++.a 
```

Comment:
I wonder if we should just auto-include these when building C++?  this will be easy to forget, and will give opaque error messages if you don't include them.